app/views/layouts/application.html.erb

```diff
@@ -15,14 +15,9 @@
     environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
     navigation_items: [
       {
-        text: t("recommended_links.index.page_title"),
-        href: recommended_links_path,
-        active: primary_navigation_area == :recommended_links,
-      },
-      {
-        text: t("controls.index.page_title"),
+        text: t("primary_navigation_areas.search"),
         href: controls_path,
-        active: primary_navigation_area == :controls,
+        active: primary_navigation_area == :search,
```

Comment:
So as the `href` points to the controls path, the active tab when you select "Search" is Controls rather than Recommended Links?

Reply:
I've tried this out locally, and that is what happens.
So when the app first loads, "Search" is the active primary link and "Recommended Links" is the active secondary link. If you click on "Search" though, the active secondary link switches to "Controls". 

Really though, I think the active secondary link should be the same in both scenarios. 

Unless you're planning to change the default link when the app is loaded?